app/helpers/template_helper.py

```diff
@@ -78,6 +78,13 @@ def legal_basis_wrapper(*args, **kwargs):
     return legal_basis_wrapper
 
 
+def get_account_url():
+    metadata = get_metadata(current_user)
+    if metadata.get('account_url'):
```

Comment:
could this be?
`return metadata.get('account_url', current_app.config['RESPONDENT_ACCOUNT_URL'])`